README.md

```diff
@@ -10,14 +10,123 @@
 
-Should cover install the rails gem with gem install rails
 
-Should cover the rails application CLI and rails new
+## Why use a framework?
 
-Should explain how to start a local rails server and what port 3000 is and how to look at your rails application in a browser.
+If you wanted to put surround sound in your house, would you go and spend years researching how to best fabricate speakers, learn how to transfer sound through wires, weld your own sound board, and create from scratch every other component that would be required to have a surround sound system in your home? Most likely not, it would be much smarter and more efficient to purchase a surround sound system from an organization that already put in all of the research and development work to create a professional system.
 
-Should explain the rails console.
+In the same way, when it comes to building a web application it would technically be possible to build out all of the functionalities needed yourself, but it's typically a better idea to leverage a system that has already spent over a decade developing the tools necessary for getting applications built.
+
+
+## What is Ruby on Rails?
+
+* **A web framework** - A web framework provides developers the tools they need in order to build applications. While every application is unique there are certain components that can be found in almost every application, such as: routing, asset management, database connections, and the list goes on. A good web framework gives developers these baseline tools so that they don't have to create the base application functionality for each new project.
+* **A Ruby Gem** - At its core, Ruby on Rails is simply a set of Ruby code libraries, and since the entire codebase is open source you have the ability to review the framework to better understand how it works.
+* **A MVC framework** - MVC stands for Model-View-Controller, this essentially means that Rails takes advantage of the popular application architecture that helps developers naturally separate concerns and organize their applications properly. This setup encourages a specific set of conventions, such as placing the logic for the application in the model files, the code flow in the controllers, and the views to simply display content to the user.
+
+## What Ruby on Rails is not
+* **A programming language** - This is one of the most common misconceptions, Ruby on Rails is not a programming language, instead it is a set of code libraries built in Ruby.
+* **A slow framework** - Due to the fact that Rails is one of the most straightforward frameworks to learn, it can lead to a number of poor coding practices from beginners. However if built properly, Rails projects can be as fast as any other framework. Furthermore, with Rails service based architecture it makes it a perfect candidate for microservice applications, which can be some of the fastest and best performing applications on the web.
```

Comment:
No need to make a change here, but this is super formal sounding. Feel free to be a bit more conversational:

```
Because Rails super straightforward to learn, it can lead to some poort coding practices from beginners. By the end of this course, you'll know what to do and what not to do though. If built properly, RAils projects can be as fast as any other framework!
```